src/olympia/lib/storage.py

```diff
@@ -0,0 +1,16 @@
+from django.contrib.staticfiles.storage import ManifestStaticFilesStorage
+
+
+class ManifestStaticFilesStorageNotMaps(ManifestStaticFilesStorage):
+    patterns = (
+        (
+            '*.css',
+            (
+                r"""(?P<matched>url\(['"]{0,1}\s*(?P<url>.*?)["']{0,1}\))""",
```

Comment:
We should document that this is copied over from Django 3.2 and why we're doing it.

Reply:
actually from Django4.1 - but compatible with django3.2 as the only changes are to name the captured groups.